libzhlgen/test/functions/mac/1.6.13/LanguageChooser.zhl

```diff
@@ -0,0 +1,7 @@
+"!4889ee480335(???????\?)":
+offsetVariable std::vector<std::string>* Global_OptionsScreen_languageList;
+
+"554889e54157415641554154534883ec484889fb488b05????????":
+cleanup __amd64 void LanguageChooser::OnRender(LanguageChooser *this);
+
+// Signatures needed - all still linux 
```

Comment:
There should be a newline at the end of the file, that's why Github shows this red icon.

You can probably configure your editor to automatically do this.